File: src/main/java/br/com/jiratorio/domain/entity/embedded/Histogram.java
package br.com.jiratorio.domain.entity.embedded;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Histogram {

    private Chart<Long, Long> chart;

    private Long median;

    private Long percentile75;

    private Long percentile90;

}

File: src/main/kotlin/br/com/jiratorio/domain/entity/embedded/Histogram.kt
package br.com.jiratorio.domain.entity.embedded

import java.io.Serializable

data class Histogram(
    val chart: Chart<Long, Int>,
    val median: Long,
    val percentile75: Long,
    val percentile90: Long
) : Serializable {

    companion object {
        private const val serialVersionUID = 1L
    }

}
